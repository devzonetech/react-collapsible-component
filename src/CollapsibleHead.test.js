import React from "react";
import { CollapsibleHead } from "./index";
import renderer from "react-test-renderer";

test("CollapsibleHead render test", () => {
    const component = renderer.create(
        <CollapsibleHead className="additionalClassForHead">
            Head title 1
        </CollapsibleHead>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    expect(tree.props.className).toEqual(
        "accordion-head additionalClassForHead"
    );
    expect(tree.props["data-show-content-above-button"]).toEqual(undefined);
});
