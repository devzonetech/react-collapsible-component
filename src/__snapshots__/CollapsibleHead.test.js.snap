// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapsibleHead render test 1`] = `
<button
  className="accordion-head additionalClassForHead"
>
  Head title 1
</button>
`;
